### PR TITLE
chore(starter): improved output and commands

### PR DESCRIPTION
**Pull request by kevin-dp**
This PR hides output from the underlying processes when running `npx create-electric-app my-app` as well as in the output of the yarn commands. Also introduces a new command `db:connect` that connects to the PG database using the `DATABASE_URL` env variable. If that variable is not defined it assumes that we're running the entire stack and it will connect to the PG that is running in Docker.

### CODE REVIEW

File: examples/starter/src/index.ts [hunk 1]
```diff
@@ -69,13 +69,22 @@ if (name) {
...
-    console.log(`Success! Your ElectricSQL app is ready at \`./${projectName}\``)
+    console.log(`⚡️ Your ElectricSQL app is ready at \`./${projectName}\``)
   }
   else {
-    console.log(`Could not install project dependencies. Nevertheless the template for your app can be found at \`./${projectName}\``)
+    console.error(Buffer.concat(errors).toString())
+    console.log(`⚡️ Could not install project dependencies. Nevertheless the template for your app can be found at \`./${projectName}\``)
   }
+  
+  console.log(`Navigate to your app folder \`cd ${projectName}\` and follow the instructions in the readme.`)
```
Comment (thruflo) [inline examples/starter/src/index.ts:89]: Nit, suggest: In the `README.md`.

File: examples/starter/template/README.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ Open a new tab in your terminal. Navigate back to the same folder. Apply the mig
 ```shell
 yarn db:migrate
 ```
+The above command applies all sql files in the migrations folder so make sure they are idempotent if you run the command multiple times.
+Or, alternatively, use proper migration tooling.
```
Comment (thruflo) [inline examples/starter/template/README.md:16]: Aggressive ;)

I suggest we reorganise this a touch: minimise the info in the primary flow. So just skip this here. Instead, as discussed with @balegas we can add a more info section at the bottom that says "for info about X, Y and Z see the <link to running the examples> page".

The primary flow is zero friction / minimum cognitive overhead to dopamine, i.e.: get the thing running with the least complexity possible.

Moving the extra stuff to the docs means we only have to maintain it once.

File: examples/starter/template/db/connect.js
Comment (thruflo) [inline examples/starter/template/db/connect.js:1]: For me, I'd like `yarn db:psql` -- but I'm happy to go with connect if you prefer for nice noun:verb consistency.

File: examples/starter/template/db/migrate.js [hunk 3]
```diff
@@ -38,15 +22,13 @@ const apply = async (fileName) => {
...
-    const fileNames = fs.readdirSync(MIGRATIONS_DIR)
-    for (const file of fileNames) {
-      await apply(file)  
+  const fileNames = fs.readdirSync(MIGRATIONS_DIR)
+  for (const file of fileNames) {
+    if (path.extname(file) === '.sql') {
+      await apply(file)
     }
   }
+  console.log('⚡️ Database is migrated.')
```
Comment (thruflo) [inline examples/starter/template/db/migrate.js:31]: Suggest `Database migrated.` I think the zap is good for the end of the initial generate command.

**Review by thruflo — APPROVED** [review]

File: examples/starter/template/db/migrate.js [hunk 4]
```diff
@@ -1,34 +1,18 @@
 const fs = require('fs')
 const path = require('path')
-const parseArgs = require('minimist')
-const shell = require('shelljs')
+const { DATABASE_URL } = require('util.js')
```
Comment (balegas) [inline examples/starter/template/db/migrate.js:3]:
```suggestion
const { DATABASE_URL } = require('./util.js')
```
Fixing this I was able to run commands with DATABASE_URL

**Review by balegas — APPROVED** [review]

File: examples/starter/template/package.json [hunk 5]
```diff
@@ -7,6 +7,7 @@
     "backend:stop": "docker compose -f backend/compose/docker-compose.yaml down --volumes",
     "electric:start": "node ./backend/startElectric.js",
     "db:migrate": "node ./db/migrate.js",
+    "db:connect": "node ./db/connect.js",
```
Comment (balegas) [inline examples/starter/template/package.json:10]: below, match "@types/node": "^16.9.1", with node version

File: examples/starter/template/package.json [hunk 6]
```diff
@@ -7,6 +7,7 @@
     "backend:stop": "docker compose -f backend/compose/docker-compose.yaml down --volumes",
```
Comment (balegas) [inline examples/starter/template/package.json:7]: Should we start the services as a daemon?

Not a strong opinion, but is a bit annoying that I do ```yarn backend:start``` and I lose control of that shell. Again, raising it, but not a strong opinion.

Comment (thruflo) [reply examples/starter/template/package.json:7]: I think it's nice to see the output.

**Comment by balegas** [comment]
@kevin-dp I noticed that the generated example code is not properly type-checked.
See https://github.com/electric-sql/electric/pull/333/files#diff-4e27ade820362150f9a28493ea449906cd1cf3d591863323f133674bb9f9b3a8R9

File: examples/starter/template/README.md [hunk 7]
```diff
@@ -12,6 +12,8 @@ Open a new tab in your terminal. Navigate back to the same folder. Apply the mig
 ```shell
 yarn db:migrate
```
Comment (balegas) [inline examples/starter/template/README.md:13]: - [ ] document ```DATABASE_URL``` in notes or elsewhere

**Pull request merged**